crates/rooch/src/commands/server/commands/start.rs

```diff
@@ -25,16 +29,42 @@ pub struct StartCommand {
 
 #[async_trait]
 impl CommandAction<()> for StartCommand {
-    async fn execute(self) -> RoochResult<()> {
+    async fn execute(mut self) -> RoochResult<()> {
+        let mut context = self.context_options.build().await?;
+        //load key address from server config
+        let key_address = if self.opt.key_address.is_none() {
+            let load_key_address_opt = context.server_config.key_address;
+            if load_key_address_opt.is_none() {
+                return Err(RoochError::KeyAddressDoesNotExistError);
+            }
+            load_key_address_opt.unwrap()
+        } else {
+            RoochAddress::from_str(self.opt.key_address.clone().unwrap().as_str()).map_err(|e| {
+                RoochError::CommandArgumentError(format!("Invalid Rooch key address String: {}", e))
+            })?
+        };
+
+        let key_keypair = context
+            .client_config
+            .keystore
+            .get_key_pair_by_key_pair_type(&key_address, KeyPairType::RoochKeyPairType)
+            .map_err(|e| RoochError::KeyAddressKeyPairDoesNotExistError(e.to_string()))?;
+
+        // Construct sequencer, proposer and relayer keypair
+        let mut server_opt = ServerOpt::new();
+        server_opt.sequencer_keypair = Some(key_keypair.copy());
+        server_opt.proposer_keypair = Some(key_keypair.copy());
+        server_opt.relayer_keypair = Some(key_keypair.copy());
```

Comment:
If the node operator wants to configure a different address for sequencer|proposer|relayer,  it's impossible in the current solution.

How about we add `sequencer_account: Option<address>`, `proposer_account: Option<address>`, `relayer_account` to the `RoochOpt`?

If the option is None, we just use the active account. And we do not need the `key_address` config.

Reply:
> If the node operator wants to configure a different address for sequencer|proposer|relayer, it's impossible in the current solution.
> 
> How about we add `sequencer_account: Option<address>`, `proposer_account: Option<address>`, `relayer_account` to the `RoochOpt`?
> 
> If the option is None, we just use the active account. And we do not need the `key_address` config.

Has remove key address from server config, and add `sequencer_account: Option<address>, proposer_account: Option<address>, relayer_account` to the RoochOpt